.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "dbt",
+    "image": "mcr.microsoft.com/devcontainers/python:1-3.10-bookworm",
+    "features": {
+        "ghcr.io/devcontainers-contrib/features/poetry:2": {},
+        "ghcr.io/devcontainers/features/terraform:1": {}
+    },
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "dbt.queryLimit": 50,
+                "git.allowNoVerifyCommit": true,
+                "python.defaultInterpreterPath": ".venv/bin/python"
+            },
+            "extensions": [
+                "innoverio.vscode-dbt-power-user"
+            ]
+        }
+    },
+    "secrets": {
+        "SNOWFLAKE_USER": {
+            "description": "Your Snowflake Username"
+        },
+        "SNOWFLAKE_PASSWORD": {
+            "description": "Your Snowflake Password"
+        },
+        "DBT_SCHEMA": {
+            "description": "The dev schema into which to build your dbt models (e.g. DBT_<YOURNAME>)"
+        }
```

Comment:
These are the secrets that you need to set in order to do dbt-related work in the codespace